diesel/src/query_dsl/belonging_to_dsl.rs

```diff
@@ -1,5 +1,44 @@
+/// Constructs a query that finds record(s) based on directional association with other record(s).
```

Comment:
> based on directional association with other record(s)

wow so many long words and generic singular/plural, it's almost a german sentence :O (I like it)